app/config/collections.php

```diff
@@ -4051,47 +4040,44 @@
                 'array' => false,
                 'filters' => ['json', 'encrypt'],
             ],
-            [
-                '$id' => ID::custom('resources'),
-                'type' => Database::VAR_STRING,
-                'format' => '',
-                'size' => Database::LENGTH_KEY,
-                'signed' => true,
-                'required' => true,
-                'default' => [],
-                'array' => true,
-                'filters' => [],
-            ],
             [
                 '$id' => ID::custom('statusCounters'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
                 'size' => 3000,
                 'signed' => true,
-                'required' => true,
-                'default' => null,
+                'required' => false,
                 'array' => false,
-                'filters' => ['json'],
+                'filters' => ['subQueryStatusCounters'],
             ],
             [
                 '$id' => ID::custom('resourceData'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
                 'size' => 131070,
                 'signed' => true,
-                'required' => true,
-                'default' => null,
+                'required' => false,
                 'array' => false,
-                'filters' => ['json'],
+                'filters' => ['subQueryResourceData'],
             ],
             [
                 '$id' => ID::custom('errors'),
                 'type' => Database::VAR_STRING,
                 'format' => '',
                 'size' => 65535,
                 'signed' => true,
+                'required' => false,
+                'array' => true,
+                'filters' => ['subQueryMigrationErrors'],
+            ],
+            [
+                '$id' => ID::custom('resources'),
+                'type' => Database::VAR_STRING,
+                'format' => '',
+                'size' => Database::LENGTH_KEY,
+                'signed' => true,
                 'required' => true,
-                'default' => null,
+                'default' => [],
```

Comment:
In order to make the diff more clean can you move the `resources` attribute to after the `credentials` attribute ?

Reply:
Done